integration-tests/scripts/install.sh

```diff
@@ -153,24 +120,57 @@ acs_integration() {
   fi
 }
 
-acs_quay_connection() {
-  #TODO: it needs to consider the following scenarios:
-  # 1. ACS is installed and Quay is installed
-  # 2. ACS is hosted on somewhere and registry is using quay.io
-  # 3. ACS is hosted on somewhere and registry is Artifactory
-  if [[ "${registry_config}" == "quay" ]]; then
-    echo "[INFO] Configure internal Quay integration with internal ACS"
-
+acs_registry_integration() {
+  local acs_central_url=""
+  local acs_central_username="admin"
+  local acs_central_password=""
+  if [[ "${acs_config}" == "hosted" ]]; then
+    acs_central_url="${ACS__CENTRAL_ENDPOINT}"
+    acs_central_password="${ACS__CENTRAL_PASSWORD}"
+  else
     acs_central_url=https://$(kubectl -n rhtap-acs get route central -o  'jsonpath={.spec.host}')
     acs_central_password=$(kubectl -n rhtap-acs get secret central-htpasswd -o go-template='{{index .data "password" | base64decode}}')
+  fi
+
+  # if registry_config is "quay", then perform the quay integration
+  if [[ "${registry_config}" == "quay" ]]; then
+    echo "[INFO] Configure Quay integration into ACS"
+
     quay_host=$(kubectl -n rhtap-quay get route rhtap-quay-quay -o  'jsonpath={.spec.host}')
```

Comment:
In case of hosted quay this value won't exists, no?

Reply:
oh, In case of hosted quay(quay.io), when ACS is newly created we need to configure hosted quay integration with ACS